tslint-config.json

```diff
@@ -88,7 +88,11 @@
     "prefer-object-spread": true,
     "radix": true,
     "restrict-plus-operands": true,
-    "strict-boolean-expressions": true,
```

Comment:
To be clear, this rule forbids things looking like : 
```
const port = Number(process.env.PORT) || config.get('port');
```
Indeed, it compares 2 expressions that are not booleans.
However, it can help javascript newcomers to avoid some mistakes.

Reply:
For me, it is a basic javascript rule that every javascript developer know, don't you think ?

Reply:
Yes ... What about checking Airbnb and Google TSLINT to know if they apply this rule ?

Reply:
- Airbnb : https://github.com/progre/tslint-config-airbnb/blob/master/tslint.js#L48
- Google did not specify this rule

Reply:
ok. Let's go for it if the others agree.